force-app/main/default/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -404,6 +411,16 @@ public with sharing class ADDR_Addresses_UTIL {
         sobjDst.put(strFieldPrefixDst + 'Latitude', sobjSrc.get(strFieldPrefixSrc + 'Latitude'));
         sobjDst.put(strFieldPrefixDst + 'Longitude', sobjSrc.get(strFieldPrefixSrc + 'Longitude'));
 
+        if (
```

Comment:
This is a good addition for a minimum change. In an ideal world, we would go a lot further here, but we are working with our hands tied. It might be worth extracting this check to a method or changing the constant name to something that indicates it is the prefix for address types that support inactive address functionality.